sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSession.java

```diff
@@ -34,11 +36,15 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
 
 /**
  * Represents an AMQP session using proton-j reactor.
  */
 public class ReactorSession implements AmqpSession {
+
+    private static final String TRANSACTION_LINK_NAME = "coordinator";
```

Comment:
Are you only allowed one transaction per session? It seems like a limitation.

Reply:
These is only one sender link with this name per session. Thats how it is implemented in track1 and in other languages also.

Reply:
Why is this controlled by a map then (line 50)? There can only be one.